config/webpack/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+
+const { join } = require('path');
+
+const shims = require('./webpack.shims');
+const basedir = join(__dirname, '../..');
+
+const config = require('../config');
+
+const prod = process.env.NODE_ENV === 'production';
+
+module.exports = merge(shims, {
+  mode: prod ? 'production' : 'development',
+  entry: require.resolve('./entries/main'),
```

Comment:
Should we keep entry files as part of the app modules structure?

Something like `/modules/core/client/app/index.js` / `boot.js` / `main.js`

Reply:
That would then output one file per client module. I think it's better to leave webpack to split the files. What is common/easy with vue-router (and others I'm sure) is to use webpack dynamic loading so the resources for each page are loaded on demand without ever having to explicitly think about it.

The current approach here is basically a hack to leave the `assets/*` files as they are.

Reply:
Gotcha, yeah Angular1 + ng-ui-router + Webpack should be popular enough combo that I'm sure someone has figured it out.

> The current approach here is basically a hack to leave the assets/* files as they are.

Makes sense! :thumbsup: